### PR TITLE
Box View changed with a new layout

### PR DESCRIPTION
**Pull request by flisowna**
## Includes:
- changes on box view 
- changes on edit box view (not finished)
- added overlays to change the number of items
- adjustment to desktop and mobile screens

## To do:
- add fields on the edit view
- add scrap and lost on the box view
- tests

## Contribute: https://trello.com/c/UjCGQV4W/929-extend-box-edit-and-details-views-for-the-new-fe

### CODE REVIEW

File: react/src/views/Box/BoxView.tsx [hunk 1]
```diff
@@ -83,23 +117,109 @@ const BTBox = () => {
...
-  const boxData = mutationStatus.data?.updateBox || data?.box;
+  const boxData = mutationLocationStatus.data?.updateBox || data?.box;
+
+  const onSubmitTakeItemsFromBox = (
+    boxFormValues: ChangeNumberOfItemsBoxData
+  ) => {
+    console.log("boxLabelIdentifier", labelIdentifier);
+    console.log("boxFormValues", boxFormValues);
+
+    if (
```
Comment (spaudanjo) [inline react/src/views/Box/BoxView.tsx:172]: reminder for next PR: we probably also want to handle the other (else) case here @flisowna

File: react/src/views/Box/BoxView.tsx [hunk 2]
```diff
@@ -83,23 +117,109 @@ const BTBox = () => {
...
     return <div>Error!</div>;
   }
 
-  const boxData = mutationStatus.data?.updateBox || data?.box;
+  const boxData = mutationLocationStatus.data?.updateBox || data?.box;
+
+  const onSubmitTakeItemsFromBox = (
+    boxFormValues: ChangeNumberOfItemsBoxData
+  ) => {
+    console.log("boxLabelIdentifier", labelIdentifier);
```
Comment (spaudanjo) [inline react/src/views/Box/BoxView.tsx:169]: Reminder for the next PR (or the one who finishes the current ticket that this PR contributes to) to remove logs and commented code

File: react/src/views/Box/BoxView.tsx [hunk 3]
```diff
@@ -83,23 +117,109 @@ const BTBox = () => {
...
+    console.log("boxFormValues", boxFormValues);
+
+    if (
+      boxFormValues.numberOfItems &&
+      boxFormValues.numberOfItems > 0 &&
+      boxData?.items
+    ) {
+      updateNumberOfItemsMutation({
```
Comment (spaudanjo) [inline react/src/views/Box/BoxView.tsx:177]: I think in a follow up PR we could try to generalise (e.g. with a generic component or custom hook) the functionality of the "Add" and "Take" features

**Review by spaudanjo — APPROVED** [review]

**Comment by HaGuesto** [comment]
[2.0 Extend box edit and details views for the new FE.](https://trello.com/c/UjCGQV4W/929-20-extend-box-edit-and-details-views-for-the-new-fe)

**Pull request merged**